src/engine/client/cl_main.cpp

```diff
@@ -302,213 +307,204 @@ void CL_WriteDemoMessage( msg_t *msg, int headerBytes )
 	FS_Write( msg->data + headerBytes, len, clc.demofile );
 }
 
-/*
-====================
-CL_StopRecording_f
 
-stop recording a demo
-====================
-*/
-void CL_StopRecord_f()
+/**
+ * If a demo is being recorderd, this stops it
+ */
+void CL_StopRecord()
 {
-	int len;
-
-	if ( !clc.demorecording )
-	{
-		Log::Notice("%s", "Not recording a demo.\n" );
-		return;
-	}
-
-	// finish up
-	len = -1;
-	FS_Write( &len, 4, clc.demofile );
-	FS_Write( &len, 4, clc.demofile );
-	FS_FCloseFile( clc.demofile );
-	clc.demofile = 0;
+    if ( !clc.demorecording )
+        return;
 
-	clc.demorecording = false;
-	Cvar_Set( "cl_demorecording", "0" );  // fretn
-	Cvar_Set( "cl_demofilename", "" );  // bani
-	Log::Notice("%s", "Stopped demo.\n" );
+    // finish up
+    int len = -1;
+    FS_Write( &len, 4, clc.demofile );
+    FS_Write( &len, 4, clc.demofile );
+    FS_FCloseFile( clc.demofile );
+    clc.demofile = 0;
+
+    clc.demorecording = false;
+    Cvar::SetValueForce(cvar_demo_status_isrecording.Name(), "0");
+    Cvar::SetValueForce(cvar_demo_status_filename.Name(), "");
+    Log::Notice("%s", "Stopped demo.\n" );
 }
 
-/*
-====================
-CL_Record_f
-
-record <demoname>
-
-Begins recording a demo from the current position
-====================
-*/
-
-static char demoName[ MAX_QPATH ]; // compiler bug workaround
-void CL_Record_f()
+class DemoStopRecordCmd: public Cmd::StaticCmd
```

Comment:
nit: The command is demo_record_stop but the function is DemoStopRecord. Can the two be consistent?